dqgen/services/queries_generator.py

```diff
@@ -77,6 +84,7 @@ def generate_from_csv(ap_file_path:pathlib.Path, output_base_dir:pathlib.Path):
     queries_output.mkdir(parents=True, exist_ok=True)
 
     processed_csv_file = read_ap_from_csv(ap_file_path)
+    validate_application_profile(application_profile_df=processed_csv_file)
```

Comment:
is the validation function updated, to reflect the latest structural changes in the AP CSVs?

Reply:
yes